app/src/main/res/values/strings.xml

```diff
@@ -171,91 +171,92 @@
     <string name="chargeprice_title">Prices</string>
     <string name="chargeprice_connection_error">Could not load prices</string>
-    <string name="chargeprice_no_compatible_connectors">None of the connectors on this charging station is compatible with your vehicle.</string>
+    <string name="chargeprice_no_compatible_connectors">No compatible connectors at this charging station.</string>
     <string name="pref_chargeprice_currency">Currency</string>
     <string name="pref_my_tariffs">My charging plans</string>
     <plurals name="pref_my_tariffs_summary">
-        <item quantity="one">(will be highlighted in price comparison)</item>
-        <item quantity="other">(will be highlighted in price comparison)</item>
+        <item quantity="one">(will be highlighted in the price comparison)</item>
+        <item quantity="other">(will be highlighted in the price comparison)</item>
     </plurals>
     <string name="chargeprice_all_tariffs_selected">all plans selected</string>
     <string name="license">License</string>
     <string name="settings_charger_data">Charging stations</string>
-    <string name="pref_data_source">Data source</string>
+    <string name="pref_data_source">Source</string>
     <plurals name="chargeprice_some_tariffs_selected">
         <item quantity="one">%d plan selected</item>
         <item quantity="other">%d plans selected</item>
     </plurals>
     <string name="unknown_operator">Unknown operator</string>
-    <string name="data_sources_description">EVMap supports multiple data sources for charging stations. Please select the one you would like to use. You can always change it later in the app\'s settings.</string>
+    <string name="data_sources_description">Please pick a source for charging stations. (Can be changed in the app settings.)</string>
     <string name="data_source_goingelectric">GoingElectric.de</string>
     <string name="data_source_openchargemap">Open Charge Map</string>
-    <string name="data_source_goingelectric_desc">Very good coverage in Germany, Austria and Switzerland and many neighboring countries. Descriptions in German. Community-maintained.</string>
-    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide coverage with varying quality. Descriptions in English or local language. Community-maintained & government open data in some countries (e.g. North America, UK, France, Norway).]]></string>
+    <string name="data_source_goingelectric_desc">Great in germannic countries. Descriptions in German. Community-maintained.</string>
+    <string name="data_source_openchargemap_desc"><![CDATA[Worldwide, with varying quality. Descriptions in English or the local language. Community-maintained and open government-data in some countries (e.g. North America, UK, France, Norway).]]></string>
     <string name="next">next</string>
-    <string name="get_started">Get started</string>
-    <string name="got_it">Got it</string>
+    <string name="get_started">OK</string>
```

Comment:
I thought it would be a bit more fun not to have the same text each time while going through the tutorial... 😅

Reply:
However thrilling a tutorial it is, my strategy is to spend less time in it.